addons/website_sale/tests/test_address.py

```diff
@@ -660,6 +660,38 @@ def test_12_recompute_taxes_on_address_change(self):
             "Tax should no longer change after order confirmation",
         )
 
+    def test_two_types_address_updates(self):
```

Comment:
We try to be more clear in the test names in case it fails we know what's the root cause
```suggestion
    def test_shop_update_address_with_use_delivery_as_billing(self):
```